docs/source/user_guide/installation/GNU-linux.rst

```diff
@@ -125,9 +125,11 @@ Currently, only GNU/Linux and macOS are supported.
 
    .. code:: bash
 
-	  pip install scikits.odes
+      brew install openblas
+      pybamm_install_odes
 
-   Assuming that SUNDIALS was installed as described :ref:`above<user-install-label>`.
+   The ``pybamm_install_odes`` command is installed with PyBaMM. It automatically downloads and installs the SUNDIALS library on your
+   system (under ``~/.local``), before installing ``scikits.odes``. (Alternatively, one can install SUNDIALS without this script and run ``pip install pybamm[odes]`` to install ``pybamm`` with ``scikits.odes``.)
```

Comment:
We should also instruct users on how to avoid failures while installing `scikits.odes` without this script. We were mentioning "Assuming that SUNDIALS was installed as described above", but if someone runs `pip install pybamm[odes]` now, it would still fail even if SUNDIALS is installed on the system because the SUNDIALS_INST environment variable would not be set. Above in the same file, we recommend running `brew install sundials` before `pip install pybamm`, so we can use the location Homebrew installs it to and ask users to set SUNDIALS_INST to that location. On an ARM-based machine for my case it is installed under `/opt/homebrew/Cellar/sundials/6.6.0`, but I think it would be under `usr/local/` on an Intel-based macOS machine. We could either check that and provide that ourselves or we could ask them to run `brew info sundials` to find the location for their case (which is how I myself found it). Note: this also shows that Homebrew and `pybamm_install_odes` install SUNDIALS under different locations (which is good because we don't want to interfere with Homebrew's files).

Perhaps adding a link to https://scikits-odes.readthedocs.io/en/stable/installation.html would also be helpful. Otherwise, we should _recommend_ the use of `pybamm_install_odes`. Please feel free to resolve this comment as necessary, though I will tag @Saransh-cpp for his input on this too.